metaflow/plugins/resources_decorator.py

```diff
@@ -28,7 +28,7 @@ class ResourcesDecorator(StepDecorator):
     disk : int, optional, default None
         Disk size (in MB) required for this step. Only applies on Kubernetes.
     memory : int, default 4096
-        Memory size (in MB) required for this step.
+        Memory size (in MiB) required for this step.
```

Comment:
given that these values are used in @kubernetes as well - would be good to ensure it's the case there too. also what about disk?

Reply:
I've not used Kubernetes before, so I wasn't sure about the three instances I left unchanged:
- `@kubernetes` `memory`
- `@kubernetes` `disk`
- `@resources` `disk` (used only for Kubernetes)

I am pretty sure this one (`@resources` `memory`) should be MiB for AWS. That may also be the case for Kubernetes, but I don't know.